ip-utils.h

```diff
@@ -176,18 +183,27 @@ void janus_network_address_string_buffer_nullify(janus_network_address_string_bu
 /*!
  * \brief Test if a given network address string buffer is null-valued
  * \param b The buffer to check
- * \return some a value which is true if the given buffer is null-valued, false otherwise.
+ * \return A positive integer if the given buffer is null-valued, 0 otherwise.
  * \see \c janus_network_address_string_buffer_nullify
  */
 int janus_network_address_string_buffer_is_null(const janus_network_address_string_buffer *b);
 
 /*!
  * \brief Extract the human readable representation of a network address from a given buffer.
  * \param b The buffer containing the given network
- * \return a pointer to the human readable representation of the network address inside the given buffer, or NULL if the buffer is invalid or NULL.
+ * \return A pointer to the human readable representation of the network address inside the given buffer, or NULL if the buffer is invalid or NULL.
  * \see \c janus_network_address_to_string_buffer
  */
-const char * janus_network_address_string_from_buffer(const janus_network_address_string_buffer *b);
+const char *janus_network_address_string_from_buffer(const janus_network_address_string_buffer *b);
+
+/*!
+ * \brief Look for an interface name to prepare a janus_network_address instance
+ * \param ifas The list of interfaces to lookup into (e.g., as returned from getifaddrs)
```

Comment:
Yep, sorry, my English still sucks at times :smile: 
In my defence, I speak an excellent Italian!